src/coverlet.core/Coverage.cs

```diff
@@ -294,9 +294,10 @@ private string GetSourceLinkUrl(Dictionary<string, string> sourceLinkDocuments,
                 string key = sourceLinkDocument.Key;
                 if (Path.GetFileName(key) != "*") continue;
 
-                string relativePath = Path.GetRelativePath(Path.GetDirectoryName(key), Path.GetDirectoryName(document));
+                if (!Path.GetDirectoryName(document).StartsWith(Path.GetDirectoryName(key) + Path.DirectorySeparatorChar))
+                    continue;
 
-                if (relativePath.Contains("..")) continue;
+                var relativePath = Path.GetDirectoryName(document).Substring(Path.GetDirectoryName(key).Length + 1);
```

Comment:
If this is indeed a functional workaround to using `Path.GetRelativePath` then we can make `coverlet.msbuild` be fully compliant with `netcoreapp2.0` and `net472` frameworks by making it target `netstandard2.0` instead